erlang.mk

```diff
@@ -6120,6 +6120,8 @@ CT_OPTS ?=
 ifneq ($(wildcard $(TEST_DIR)),)
 ifndef CT_SUITES
 CT_SUITES := $(sort $(subst _SUITE.erl,,$(notdir $(call core_find,$(TEST_DIR)/,*_SUITE.erl))))
+SLOW_CT_SUITES := $(sort cluster_rename clustering_management dynamic_ha eager_sync health_check partitions priority_queue simple_ha queue_master_location unit_inbroker_backing_queue)
+FAST_CT_SUITES := $(filter-out $(SLOW_CT_SUITES),$(CT_SUITES))
```

Comment:
The implementation of `ct-slow` and `ct-fast` can also go to one of our Erlang.mk plugins so we could define `$(CT_SLOW)` and `$(CT_FAST)` in other projects too.